services/migrations/migrate.go

```diff
@@ -172,6 +172,11 @@ func newDownloader(ctx context.Context, ownerName string, opts base.MigrateOptio
 	return downloader, nil
 }
 
+// InternalMigrateRepository function
+func InternalMigrateRepository(downloader base.Downloader, uploader base.Uploader, opts base.MigrateOptions) error {
```

Comment:
Could the `migrateRepository` exposed as `MigrateRepository` directly? `InternalMigrateRepository` doesn't seem to help much.

Reply:
It was a mistake to expose InternalMigrateRepository indeed. The test was reworked so that it is not necessary.